openmct.js

```diff
@@ -23,15 +23,9 @@
 
 const matcher = /\/openmct.js$/;
 if (document.currentScript) {
-    let src = document.currentScript.src;
-    if (src && matcher.test(src)) {
-        // eslint-disable-next-line no-undef
-        __webpack_public_path__ = src.replace(matcher, '') + '/';
-    }
+    let e = document.currentScript.src;
+    e && matcher.test(e) && (__webpack_public_path__ = e.replace(matcher, "") + "/")
```

Comment:
We do not need to minify source code since that is done with webpack. The goal of the source code is to be readable and understandable.We do not need to minify source code since that is done with webpack. The goal of the source code is to be readable and understandable. This change makes it harder to understand what is going on and we should stay away from that.